clients/mockStoreClient.go

```diff
@@ -30,6 +30,12 @@ func (d *MockStoreClient) UpsertConfirmation(notification *models.Confirmation)
 	if d.doBad {
 		return errors.New("UpsertConfirmation failure")
 	}
+	if notification.Email == "patient@myemail.com" && notification.Key != "" && notification.ShortKey != "" {
```

Comment:
I don't understand the logic. This mock will always return nil...?

Reply:
if it does not return nil, it means that key and shortKey haven't been generated; That's the objective of the test: validate both keys are generated for a patient call.